### ISSUE TITLE
[Bug]: Typing indicator doesn't appear on long running messages

### ISSUE THREAD

**Issue by therealjohn** [microsoft/teams-ai#1275]
### Language

Javascript/Typescript

### Version

latest

### Description

I don't see a typing indicator while the bot is responding, even after a few seconds. 

I expect it to work following this logic: https://github.com/microsoft/teams-ai/blob/09ebba139104816ea675e6e7958e479c7c3de6d2/js/packages/teams-ai/src/Application.ts#L635

### Reproduction Steps

```bash
1. Create a AI Chat bot project using Teams Toolkit
2. Fill in the OpenAI Key in .env.local.user
3. F5 to Teams
4. Ask a question
5. It takes about 3-5 seconds for the OpenAI response, meanwhile there is no typing indicator. The response just appears.
```
[TestAIChatBot2.zip](https://github.com/microsoft/teams-ai/files/14213824/TestAIChatBot2.zip)

Here's a video showing the issue:

https://github.com/microsoft/teams-ai/assets/1501196/e245cf4a-670e-4057-b6ac-e0d8d549628a

**Comment by Stevenic** [comment]
Interesting… the type timer should start automatically. This would imply that some intermediate message was sent causing it to stop.

**Comment by singhk97** [comment]
Hi @therealjohn, It's a Teams client issue. I had just set up a bot that uses the built-in "start typing timer" functionality that comes with the vanilla `Application` setup. What I found was that sometimes it wouldn't show and sometimes it would. There's also a case where it would show after the bot sends a message, which is quite weird. 

It works in the Teams V2 desktop client, but quite inconsistent.


https://github.com/microsoft/teams-ai/assets/115390646/ac24dade-4c61-4974-a250-550da172f8bb
